redaxo/src/addons/install/lib/api/api_core_update.php

```diff
@@ -202,6 +204,8 @@ public function execute()
             }
             rex_package_manager::generatePackageOrder();
 
+            rex_logger::factory()->info('REDAXO Core updated to version '. $newCoreVersion);
```

Comment:
Ich fände nicht verkehrt, auch die vorherige Version zu erwähnen, also `from X to Y`.

Reply:
done